src/Tribe/Views/V2/Hooks.php

```diff
@@ -95,6 +96,20 @@ protected function add_filters() {
 		}
 	}
 
+	/**
+	 * Includes includes edge cases for filtering when we need to manually overwrite theme's read
+	 * more link when excerpt is cut programatically.
+	 *
+	 * @see   tribe_events_get_the_excerpt
+	 *
+	 * @since 4.9.11
+	 *
+	 * @return void
+	 */
```

Comment:
:no_entry_sign: Function return type is void, but function contains return statement

<sub>hash: 47f566c8305086b17277c198623c62a9</sub>